### PR TITLE
No README message

### LINKED ISSUE #244: No README? Then say it.
Upload a module with no README and stare 10 mins at this:
image

### PR DESCRIPTION
**Pull request by martonlederer**
Closes #244

### CODE REVIEW

**Comment by martonlederer** [comment]
#286 should be merged before this

File: vue.config.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ module.exports = {
     markdownRule.test(/\.md/).use("raw-loader").loader("raw-loader");
   },
   devServer: {
-    // proxy: "https://nest.land",
+    proxy: "https://nest.land",
```
Comment (t8) [inline vue.config.js:19]: Do we need this?

**Review by t8 — APPROVED** [review]
LGTM

This was so overdue. Thanks for adding this!

File: vue.config.js (hunk 1 above)
Comment (martonlederer) [reply vue.config.js]: Doesn't matter if it is there, it is not uses in production

**Pull request merged**